CoreDataAccessView/test/unit/src/au/gov/asd/tac/constellation/views/dataaccess/panes/DataAccessPaneNGTest.java

```diff
@@ -409,15 +417,45 @@ public void qualityControlRuleChanged_cant_run() {
         dataAccessPane.qualityControlRuleChanged(false);
 
         verify(buttonToolbar).changeExecuteButtonState(ButtonToolbar.ExecuteButtonState.CALCULATING, true);
+    }   
+
+    @Test
+    public void testCreateCombo() throws Exception {
+        final DataAccessPane dataAccessPane = mock(DataAccessPane.class);
+        when(dataAccessPane.createCombo(any())).thenCallRealMethod();
+
+        KeyEvent keyEvent = new KeyEvent(KeyEvent.KEY_PRESSED, "Ctrl", "A", KeyCode.A, false, true, false, false);
+        KeyCombination keyCombination = dataAccessPane.createCombo(keyEvent);
+        assertTrue(keyCombination != null);
+        assertTrue(keyCombination.getDisplayText().equals("Ctrl+A"));
+
+        KeyEvent keyEvent1 = new KeyEvent(KeyEvent.KEY_PRESSED, "Shift", "A", KeyCode.A, true, false, false, false);
+        KeyCombination keyCombination1 = dataAccessPane.createCombo(keyEvent1);
+        assertTrue(keyCombination1 != null);
+        assertTrue(keyCombination1.getDisplayText().equals("Shift+A"));
+
+        KeyEvent keyEvent2 = new KeyEvent(KeyEvent.KEY_PRESSED, "Alt", "A", KeyCode.A, false, false, true, false);
+        KeyCombination keyCombination2 = dataAccessPane.createCombo(keyEvent2);
+        assertTrue(keyCombination2 != null);
+        assertTrue(keyCombination2.getDisplayText().equals("Alt+A"));
+
+        KeyEvent keyEvent3 = new KeyEvent(KeyEvent.KEY_PRESSED, "Meta", "A", KeyCode.A, false, false, false, true);
+        KeyCombination keyCombination3 = dataAccessPane.createCombo(keyEvent3);
```

Comment:
All the KeyEvent and KeyCombination variables in this file can be `final` (there are 8 of them)

Reply:
done